README.md

```diff
@@ -1,3 +1,6 @@
+[![NuGet version](https://badge.fury.io/nu/Microsoft.DevTunnels.Ssh.svg)](https://badge.fury.io/nu/Microsoft.DevTunnels.Ssh)
```

Comment:
Shouldn't that be version of dev tunnels packages, not dev tunnels ssh?

Reply:
hmm, what would be the package name of it. ? I checked the npm and nuget Microsoft dev tunnels package lists and couldn't find anything that indicates sdk.

Reply:
When we make a change in sdk repo it would change the Microsoft.DevTunnels.Connections package but it does not have any reference in here.

Reply:
changed as dev-tunnels-connections package